### ISSUE TITLE
Provide controls to disable the LEDs

### ISSUE THREAD

**Issue by drothenberger** [roomsense/firmware#6]
Please provide controls to disable the LEDs (both for RoomSense and the ClimateSense attachment), so the device can be used in a bedroom or other dark space (such as media room) where the flashing lights would be distracting.

Ideally, each of the lights would be exposed to Home Assistant, so HA automations can turn them on and off.

**Comment by marcel-wiedemeier** [comment]
Agreed. At least allow disabling the occupied light

**Comment by dwwkelly** [comment]
Yes, controlling the LED status by time of day would be great.  I'd take just the option to turn it off though.

**Comment by roomsense** [comment]
We added this feature in the firmware [20240903](https://github.com/roomsense/firmware/tree/main/latest)

**Comment by drothenberger** [comment]
Thanks!

Would it be possible to expose these controls to Home Assistant, so the LEDs can be controlled through automations? For example, I'd like to turn off the LEDs at night and turn them on in the morning.

**Comment by russellmilliner** [comment]
maybe even control the color of the LEDs in HA??

**Comment by roomsense** [comment]
will consider it in the future updates.
